myframe.cpp

```diff
@@ -2735,27 +2735,38 @@ void MyFrame::RegisterTextCtrl(wxTextCtrl *ctrl)
     ctrl->Bind(wxEVT_KILL_FOCUS, &MyFrame::OnTextControlKillFocus, this);
 }
 
-// Reset the guiding parameters and the various graphical displays when binning changes.  This should be done when guiding starts
-// so the user can experiment with binning without losing guider settings
-void MyFrame::HandleBinningChange()
+// Reset the guiding parameters and the various graphical displays when image scale is changed outside the AD UI.  Goal is to restore basic guiding behavior
+// until a fresh calibration is done.  Ratio of image scales is used because 1) info like mount guide speed may not be available and 2) the user may have already 
+// adjusted the calibration step-size in the profile to get the results he wants.
+void MyFrame::HandleImageScaleChange(double NewToOldRatio)
 {
     if (pMount)
     {
+        // Adjust the calibration step-size to facilitate a reasonable calibration
+        if (!pMount->IsStepGuider())

```

Comment:
I think this should be `Scope *scope = TheScope(); if (scope) { int stepSize = scope->GetCalibrationDuration(); ... scope->SetCalibrationDuration(...`
so that it updates the scope calibration step size for the scope when the primary "mount" is the ao.